lib/src/main/java/com/squareup/cycler/RecyclerMutations.kt

```diff
@@ -1,24 +1,55 @@
 package com.squareup.cycler
 
+import android.graphics.Canvas
 import android.view.MotionEvent
 import android.view.View
+import android.view.View.MeasureSpec
+import androidx.annotation.VisibleForTesting
 import androidx.recyclerview.widget.ItemTouchHelper
 import androidx.recyclerview.widget.RecyclerView
 import androidx.recyclerview.widget.RecyclerView.ViewHolder
+import com.squareup.cycler.Recycler.CreatorContext
+import com.squareup.cycler.Recycler.RowSpec
 import com.squareup.cycler.StandardRowSpec.Creator
+import com.squareup.cycler.SwipeDirection.END
+import com.squareup.cycler.SwipeDirection.LEFT
+import com.squareup.cycler.SwipeDirection.RIGHT
+import com.squareup.cycler.SwipeDirection.START
+import java.util.EnumSet
+import kotlin.math.abs
 
 fun <I : Any> Recycler.Config<I>.enableMutations(block: MutationExtensionSpec<I>.() -> Unit) {
   extension(MutationExtensionSpec<I>().apply(block))
 }
 
+/**
+ * Used in swipe related methods. They can be absolute or text-direction relative ([START]/[END]).
```

Comment:
Why do we need to support both left/right and start/end? Seems like it would be better to just force consumers to be RTL-compatible by only providing start/end APIs.

Reply:
I'm not sure we want to be always RTL dependent. Maybe we want absolute directions for the swipe gesture. That's why I left both.